### PR TITLE
Support inspecting an exec command.

### PR DESCRIPTION
**Pull request by mbruggmann**
This is handy to check the exit code of a previously run docker
exec command.

### CODE REVIEW

**Comment by rohansingh** [comment]
:+1:

**Comment by rohansingh** [comment]
Want to rebase this on master? Just to make sure that no additional tests fail other than the known failures.

**Comment by rohansingh** [comment]
Might I bother you to rebase one last time? :-)

File: src/main/java/com/spotify/docker/client/DefaultDockerClient.java [hunk 1]
```diff
@@ -996,6 +997,22 @@ public LogStream execStart(String execId, ExecStartParameter... params)
     }
   }
 
+  @Override
+  public ExecState execInspect(String execId) throws DockerException, InterruptedException {
```
Comment (rohansingh) [inline src/main/java/com/spotify/docker/client/DefaultDockerClient.java]: `final String execId`

**Comment by mbruggmann** [comment]
@rohansingh I only pressed rebuild 9 times, but it worked now.

**Pull request merged**